libplatsupport/mach_include/spacemit/platsupport/mach/timer.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2025, 10xEngineers 
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#pragma once
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#define SPACEMIT_TIMER_BASE      0xD4014000
+#define SPACEMIT_TIMER_SIZE      0x2000u
+#define SPACEMIT_TIMER0_IRQ      0x17u
+#define SPACEMIT_TIMER1_IRQ      0x1Au
```

Comment:
I generally follow the datasheet notation, as that makes it easier to check.

0x17 = 23 = timer0_1_irq
0x1A = 26 = timer1_1_irq

Don't you want to use the same peripheral and use timer0_2_irq (24) instead?